### ISSUE TITLE
Need help to develop?

### ISSUE THREAD

**Issue by thyarles** [hssm/media-import#18]
I saw a long time without updates. If you need a hand, let me know. Would be nice if we can choose the Deck to import the media on, it is not an issue as we can easily change it later, but we can improve it. Thanks.

**Comment by FeatheredHwang** [comment]
Hey Charles,
Thanks for your hand. I’m sorry that I have such a long time that not using Anki anymore. Anki is wonderful but I’m too lazy to carry it on. You can folk my GitHub code if you like. Hope the code helps. Hope I can return to Anki’s Family again!

BEST WISHES!

Your sincerely,
Kyle

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

From: Charles ***@***.***>
Sent: Saturday, September 9, 2023 1:46 PM
To: ***@***.***>
Cc: ***@***.***>
Subject: [hssm/media-import] Need help to develop? (Issue #18)


I saw a long time without updates. If you need a hand, let me know. Would be nice if we can choose the Deck to import the media on, it is not an issue as we can easily change it later, but we can improve it. Thanks.

**Comment by petecodes100** [comment]
> I saw a long time without updates. If you need a hand, let me know. Would be nice if we can choose the Deck to import the media on, it is not an issue as we can easily change it later, but we can improve it. Thanks.

Hello, are you considering to update this plugin? Here are some infos on how to make it compatible again: https://forums.ankiweb.net/t/porting-tips-for-anki-23-10/35916 Unfortunately, I don't know how the code would need to be changed.

**Comment by thyarles** [comment]
> > I saw a long time without updates. If you need a hand, let me know. Would be nice if we can choose the Deck to import the media on, it is not an issue as we can easily change it later, but we can improve it. Thanks.
> 
> Hello, are you considering to update this plugin? Here are some infos on how to make it compatible again: https://forums.ankiweb.net/t/porting-tips-for-anki-23-10/35916 Unfortunately, I don't know how the code would need to be changed.

Hi @petecodes100... I thought that I was the only one interested on this, so I solved on my local machine. Soon as I have some time (late December, I guess), I'll propose an update here. Thanks for replying.

**Comment by badrange** [comment]
> Hi @petecodes100... I thought that I was the only one interested on this, so I solved on my local machine. Soon as I have some time (late December, I guess), I'll propose an update here. Thanks for replying.

I also use this plugin, I'm sure there are quite a few users who are not using GitHub too

**Comment by petecodes100** [comment]
> > > I saw a long time without updates. If you need a hand, let me know. Would be nice if we can choose the Deck to import the media on, it is not an issue as we can easily change it later, but we can improve it. Thanks.
> > 
> > 
> > Hello, are you considering to update this plugin? Here are some infos on how to make it compatible again: https://forums.ankiweb.net/t/porting-tips-for-anki-23-10/35916 Unfortunately, I don't know how the code would need to be changed.
> 
> Hi @petecodes100... I thought that I was the only one interested on this, so I solved on my local machine. Soon as I have some time (late December, I guess), I'll propose an update here. Thanks for replying.

Yes, I think many people would be happy to see this plugin work again! Really maked learning with videos/multimedia much easier.
